CODEOWNERS

```diff
@@ -13,4 +13,8 @@
 /src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md @chrisgorgo
 /src/04-modality-specific-files/03-electroencephalography.md @sappelhoff @ezemikulan
 /src/04-modality-specific-files/04-intracranial-electroencephalography.md @ezemikulan
+/src/05-derivatives/03-imaging.md @effigies
+/src/05-derivatives/04-structural-derivatives.md @edickie @ahoopes
+/src/05-derivatives/05-functional-derivatives.md @effigies
+/src/05-derivatives/06-diffusion-derivatives.md @francopestilli @oesteban
```

Comment:
What's the policy around "code ownership"? I've already taken a large hatchet to the DWI derivatives and plan to do a lot more, so would be happy to be listed as an "owner" alongside @francopestilli and @oesteban.

Reply:
```suggestion
/src/05-derivatives/06-diffusion-derivatives.md @francopestilli @oesteban @Lestropie
```

Reply:
This is fine for me @effigies @Lestropie @oesteban